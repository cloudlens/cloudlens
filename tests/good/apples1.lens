{
    totalApples = 0;
}

match{
    "(?<numApples>\d+) apples"
}

stream{
    totalApples += parseInt(e.numApples);
    if(totalApples > 10) print("We picked too many apples!");
}
