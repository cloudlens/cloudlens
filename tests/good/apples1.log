Alice picked 3 apples
Nicolas picked more: 4 apples
Olivier picked the most: 5 apples
