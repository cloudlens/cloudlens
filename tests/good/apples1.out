We picked too many apples!
